plutus-playground/plutus-playground-server/src/Playground/Interpreter.hs

```diff
@@ -190,18 +221,7 @@ mkApplyExpr functionName [a, b, c, d, e, f] =
     "apply6" <+> functionName <+> a <+> b <+> c <+> d <+> e <+> f
 mkApplyExpr functionName [a, b, c, d, e, f, g] =
     "apply7" <+> functionName <+> a <+> b <+> c <+> d <+> e <+> f <+> g
+mkApplyExpr _ _ = error "cannot apply more than 7 arguments"
```

Comment:
This seems bad - won't this let people crash the playground server?

Reply:
no, just the thread, why would it crash the server?

7 is an arbitrary number, we can increases it if someone wants to painfully construct 8, 9, 10 etc.

Reply:
Ah okay, I'm not too clear on how this stuff works. Just wondering if we should try and push `MonadError` down to here, although that's probably a hassle.

Reply:
Oh yeah, I see, yeah I think we don't want to because this is a function that's actually evaluated by the interpreter.